### DISCUSSION TITLE
Texture stripes cause artifacts

### DISCUSSION THREAD

**Discussion by deers** [google/model-viewer#2763] [Q&A] [answered]
### Description

The texture looks different on my mobile browser, I used chrome on nexus 6p and the texture look like this:
 ![2021-09-06](https://user-images.githubusercontent.com/814376/132177988-67af550e-4e48-42c0-bedb-7ec6106d1997.png)
it should look like this:
![Capture](https://user-images.githubusercontent.com/814376/132178211-d61418f9-f5b8-4eb3-ae70-08c0e2e01aa8.JPG)
do you know how to fix this issue?

#### Live Demo

https://texturetest.glitch.me/

### Version

- model-viewer: https://unpkg.com/@google/model-viewer/dist/model-viewer.js

### Browser Affected

- [x] Chrome, version: 92.0.4515.166
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by elalish** [comment, marked as answer]
I can see this behavior on regular desktop Chrome as well. This is a result of the way we anti-alias your texture (which has many fine stripes, and so is particularly aliasing prone). You are seeing different mipmap levels on different parts of the sphere, which end up colored a bit differently. You might be able to improve this by tweaking your texture slightly; the colors outside of your UV mapping matter here, for instance. It might also help to make your stripes a power-of-2 pixels wide to help with consistent mipmapping. But in general, tiny stripes in textures will tend to make your life difficult.

**Reply by deers** [reply]
Thank you, making the stripes a power of 2 pixels is solving the problem.
